### PR TITLE
API-413: Update README for isAgeVerified() bug

### PR DESCRIPTION
**Pull request by bucky-boy**
Is this a reasonable update for isAgeVerified()?

I'll bump the version numbers to 1.4.2 when it has been released.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ If you are using Maven, you need to add the following dependency:
 
 If you are using Gradle, here is the dependency to add:
 
-`compile group: 'com.yoti', name: 'yoti-sdk-impl', version: '1.3'`
+`compile group: 'com.yoti', name: 'yoti-sdk-impl', version: '1.4.1'`
```
Comment (qzagarese) [inline README.md]: we are re-updating this once we release the fix, right?

File: README.md [hunk 2]
```diff
@@ -302,6 +303,20 @@ Combining this with the Spring Boot Auto Configuration can make integration very
...
 
+### Age Verification
+
+#### Affects
+
+* Versions prior to 1.4.2
+
+#### Description
+
+A bug in the isAgeVerified() property of the profile meant that isAgeVerified() would always return null (i.e. not checked).
```
Comment (vassyz) [inline README.md]: Could you please format the method name like this `isAgeVerified()`?

**Comment by vassyz** [comment]
Looks good to me. Is it possible to add `isAgeVerified` to the example?

File: README.md (hunk 1 above)
Comment (bucky-boy) [reply README.md]: Yes.  As soon as 1.4.2 is out there, I will update the readme.

File: README.md (hunk 2 above)
Comment (bucky-boy) [reply README.md]: Sure

**Comment by bucky-boy** [comment]
If you mean add it to the spring example, that's a good idea.  For now I'd rather get this fix deployed, then add that in when we get an opportunity to test it (that is, test the spring example too).

**Pull request merged**